components/NavBar/NavBar.test.js

```diff
@@ -0,0 +1,16 @@
+import { render, screen } from "@testing-library/react";
+import NavBar from "./NavBar";
+
+jest.mock("next/router", () => ({
+  router() {
+    return { asPath: "/" };
+  },
+}));
+
+test("navbar link to homepage", () => {
+  render(<NavBar />);
+  const navagationLinks = screen.getAllByRole("link");
+
+  expect(navagationLinks.length).toBe(1);
```

Comment:
```suggestion
  expect(navagationLinks).toHaveLength(1);
```
But if there is only one, you should rather use `getByRole` and test for to being in the document.